web/src/hooks/queries/useStandardize.ts

```diff
@@ -0,0 +1,19 @@
+import { useQuery } from '@tanstack/react-query';
+
+import { getStandardizedCols } from '../../api/project';
+import { useSession } from '../../contexts/session-context';
+
+export const useStandardize = (
+  namespace: string | undefined,
+  project: string | undefined,
+  tag: string | undefined,
+  schema: string | undefined,
+) => {
+  const session = useSession();
+  const query = useQuery({
+    queryKey: [namespace, project, tag],
```

Comment:
```suggestion
queryKey: ['standardize', schema, namespace, project, tag],
```
Can we change the key for this so it gets cached properly?